framegear/components/Frame/Frame.tsx

```diff
@@ -40,8 +40,8 @@ function ValidFrame({ metadata }: { metadata: FrameMetadataWithImageObject }) {
     <div>
       {/* eslint-disable-next-line @next/next/no-img-element */}
       <img
-        className={`w-full rounded-t-xl ${imageAspectRatioClassname} object-cover`}
-        src={image.src}
+        className={`w-full rounded-t-xl ${imageAspectRatioClassname} ${image.src ? "object-cover" : "object-contain"}`}
+        src={image.src ? image.src : "https://demofree.sirv.com/nope-not-here.jpg?w=750"}
```

Comment:
Sorry we can not have links here on random websites.

Reply:
would it be acceptable if I add the images directly to the public folder of the project and reference them from there?

Reply:
yup, but only if it's a cool image. If it's not, is probably best have something else as default.

Reply:
![image](https://github.com/user-attachments/assets/e3e10b28-8d61-4f46-b1a5-3f1494c54972)
 is it ok ?

Reply:
I think is best we avoid add those kind of images so far. As we want our designer to take a look at it.

Reply:
Ok, I'll update the image based on the designer's review.